src/components/Products/Products.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react';
+import Filters from './Filters';
+import useHttpRequest from '../../hooks/useHttpRequest';
+import { useSearchParams } from 'react-router-dom';
+
+const Products = () => {
+  const [searchParams] = useSearchParams();
+
+  const name = searchParams.get('name') || '';
+  const category = searchParams.get('category') || '';
+  const sizes = searchParams.get('sizes') || '';
+  const minPrice = searchParams.get('minPrice') || '';
+  const maxPrice = searchParams.get('maxPrice') || '';
+
+  const {
+    fetchRequest,
+    state: { error, data, loading }
+  } = useHttpRequest({
+    method: 'GET',
+    url: `products/?name=${name}&category=${category}&sizes=${sizes}&minPrice=${minPrice}&maxPrice=${maxPrice}`
+  });
+
+  useEffect(() => {
+    fetchRequest();
+  }, [category, sizes, minPrice, maxPrice]);
```

Comment:
Shouldn't name be here too? You can also write just [searchParams] instead of all separately

Reply:
Fixed. Thanks!